screen-capture/META.yml

```diff
@@ -0,0 +1,5 @@
+spec: https://w3c.github.io/mediacapture-screen-share/
+suggested_reviewers:
+  - alvestrand
+  - martinthomson
+  - suhasHere
```

Comment:
@suhasHere, can you confirm that you're OK with getting reviews for this test suite, and that you've accepted the org invite? Thanks!